src/_algopy_testing/arc4.py

```diff
@@ -1053,8 +1053,8 @@ class Struct(MutableBytes, _ABIEncoded, metaclass=_StructMeta):  # type: ignore[
 
     _type_info: typing.ClassVar[_StructTypeInfo]  # type: ignore[misc]
 
-    def __init_subclass__(cls) -> None:
-        dataclasses.dataclass(cls)
+    def __init_subclass__(cls, **kwargs: dict[str, typing.Any]) -> None:
```

Comment:
should accept and forward on *args as well

Reply:
@daniel-makerx good point, pushed the changes